### PR TITLE
feat (#664): Renaming Analyst app to Research Repository

### PR DESCRIPTION
**Pull request by ghanlohar**
# Closes/Fixes/Resolves
Resolves #664

# Explain the feature/fix
Renamed Analyst SPA to Research Repository in the hosted documentation for Analyst app.

## Does this PR introduce a breaking change
No

### CODE REVIEW

**Review by sayak-sarkar — APPROVED** [review]

**Review by sayak-sarkar — CHANGES_REQUESTED** [review]
@ghanlohar Can you please rename the document name and update the path on the [sidebars.js](https://github.com/1-Platform/one-platform/blob/master/packages/documentation-spa/sidebars.js) file as well?

File: packages/documentation-spa/docs/apps/hosted/research-repository/research-repository-spa.md [hunk 1]
```diff
@@ -91,17 +89,17 @@ Analyst Papers Microservice provides a set of GraphQL Queries and Mutation APIs
...
 
-    The Homepage shows the documents from a cache stored in the Analyst Papers backend. The cache is updated every day to check for new document additions/removals.
+    The Homepage shows the documents from a cache stored in the Research Repository backend. The cache is updated every day to check for new document additions/removals.
 
     But unlike the homepage, newly added documents are searchable almost immediately. So they show up in search even though it doesn’t show up on the homepage.
 
-* **Analyst Papers Architecture**
-    ![img](./analyst-papers-architecture.svg)
+* **Research Repository Architecture**
+    ![img](./research-repository-architecture.svg)
```
Comment (deshmukhmayur) [inline packages/documentation-spa/docs/apps/hosted/research-repository/research-repository-spa.md:105]: Can you also change the "Analyst Papers Cache" to "Research Papers Cache" in the architecture diagram?

The editable diagram is in the Google Doc. Just rename it and save it as an image.

**Review by deshmukhmayur — CHANGES_REQUESTED** [review]

**Review by sayak-sarkar — APPROVED** [review]

File: packages/documentation-spa/docs/apps/hosted/research-repository/research-repository-spa.md (hunk 1 above)
Comment (sayak-sarkar) [reply packages/documentation-spa/docs/apps/hosted/research-repository/research-repository-spa.md:105]: Good catch!

**Comment by sayak-sarkar** [comment]
Looks good to be merged once the images are updated.

**Review by deshmukhmayur — APPROVED** [review]

**Pull request merged**